libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -122,12 +122,17 @@ class AP_ICEngine {
 
     enum class Options : uint16_t {
         DISABLE_IGNITION_RC_FAILSAFE=(1U<<0),
+        REDLINE_WARNING = (1U << 0),
     };
     AP_Int16 options;
 
     // start_chan debounce
     uint16_t start_chan_last_value = 1500;
     uint32_t start_chan_last_ms;
+
+    // redline rpm
+    AP_Int16 redline_rpm;
+    bool redline_flag = false;
```

Comment:
```suggestion
    bool redline_flag;
```
setting to false is just a waste of flash. It's already 0 when constructed.

Reply:
Fixed. Thank you!